conf/deployment/baremetal/ai_1az_rhcos_nvme_intel_3m_3w.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetal'
+  deployment_type: 'ai'
```

Comment:
I hope that using ai as deployment type won't be confusing but it makes sense to shorten assisted installer.